spec/features/challenge_partnership_page_spec.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.feature "Reporting an error with a partnership", type: :feature, js: true, rutabaga: false do
+  let!(:cohort) { create :cohort, start_year: 2021 }
+
+  let!(:privacy_policy) do
+    create :privacy_policy
+    PrivacyPolicy::Publish.call
+  end
+
+  describe "when using an email link" do
+    scenario "Can see challenge options from an email link" do
+      given_a_fip_school_with_a_partnership_that_can_be_challenged "test-sit@example.com", "Test school", "111111-test-school", "abc1234"
```

Comment:
I don't think you need to input these variables at all because they are all the same — can you just put them in the method?